test/e2e_node/jenkins/benchmark/jenkins-benchmark.properties

```diff
@@ -0,0 +1,9 @@
+GCE_HOSTS=
+GCE_IMAGE_CONFIG_PATH=test/e2e_node/jenkins/benchmark/benchmark-config.yaml
+GCE_ZONE=us-central1-f
+GCE_PROJECT=k8s-jkns-ci-node-e2e
+CLEANUP=true
+GINKGO_FLAGS='--skip="\[Flaky\]"'
+SETUP_NODE=false
```

Comment:
`SETUP_NODE` is used to put current user into docker user group https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/e2e_remote.go#L157 so as to avoid using `sudo`.

Because we've already put jenkins into the docker user group when building the test image, so we don't need to `SETUP_NODE` in jenkins job.

My future PR will make sure the whole test suite is running by root, at that time the option is never needed anymore.